requirements.txt

```diff
@@ -1,5 +1,5 @@
 ovos-config>=0.0.12,<1.0.0
 ovos-utils>=0.0.38,<1.0.0
 websocket-client>=0.54.0
-pyee>=8.1.0, < 9.0.0
+pyee>=12.0.0
```

Comment:
I think this is backwards compatible, probably don't need to force a newer version

Reply:
it's not, [old import no longer available](https://github.com/jfhbrook/pyee/blob/main/pyee/__init__.py)

## 2024/08/30 Version 12.0.0

- Remove deprecated imports:
  - `pyee.BaseEventEmitter`
  - `pyee.AsyncIOEventEmitter`
  - `pyee.TwistedEventEmitter`
  - `pyee.ExecutorEventEmitter`
  - `pyee.TrioEventEmitter`
- Add `PyeeError` which inherits from `PyeeException`, and use throughout
- Deprecate direct use of `PyeeException`

Reply:
he, seems the imports are completely gone, not even just moved around...

Reply:
https://github.com/OpenVoiceOS/ovos-bus-client/pull/124 with backwards compat we can only go up to 11.1.1